### ISSUE TITLE
Oxidized with local git?

### ISSUE THREAD

**Issue by Ondjultomte** [furhouse/puppet-oxidized#2]
Does it also install and config a local git repo?

**Comment by furhouse** [comment]
No, this module does not install git. It would make sense to install it, though. In the meanwhile, I'll make sure to include it in the Limitations of the README.

When git is installed, https://github.com/ytti/oxidized/blob/master/docs/Outputs.md#output-git should take care of configuring the local repository.
